frontend/src/component/admin/network/NetworkTrafficUsage/NetworkTrafficUsage.tsx

```diff
@@ -248,49 +269,74 @@ const NewNetworkTrafficUsage: FC = () => {
         }
     }, [data]);
 
+    // todo: extract this (and also endpoints info)
+    const [lastLabel, ...otherLabels] = Object.values(endpointsInfo)
+        .map((info) => info.label.toLowerCase())
+        .toReversed();
+    const requestTypes = `${otherLabels.toReversed().join(', ')}, and ${lastLabel}`;
+
     return (
         <ConditionallyRender
             condition={isOss()}
             show={<Alert severity='warning'>Not enabled.</Alert>}
             elseShow={
                 <>
                     <ConditionallyRender
-                        condition={includedTraffic > 0 && overageCost > 0}
+                        condition={
+                            (newPeriod.grouping === 'monthly' ||
+                                newPeriod.month ===
+                                    format(new Date(), 'yyyy-MM')) &&
```

Comment:
There's a `currentPeriod` in `useTrafficDataEstimation`, but maybe that's not available here?

Reply:
Ah, good call. Should be actually!

Reply:
Mmm ... So, there **is**, but that variable has: the current date and a label 'current month'. Neither of those really help out much here. We'd have to take the value from that variable and format that. It's probably slightly better than instantiating a new date object every time, but only marginally.

I'd rather extract this into a separate function or variable (that might be tied to the selection object). But I'd like to look at that separately too, if you're alright with it.